src/activemasternode.cpp

```diff
@@ -75,8 +61,11 @@ void CActiveMasternode::ManageStatus()
         }
 
         // The service needs the correct default port to work properly
-        if (!CMasternodeBroadcast::CheckDefaultPort(service, errorMessage, "CActiveMasternode::ManageStatus()"))
+        std::string errorMessage;
+        if (!CMasternodeBroadcast::CheckDefaultPort(service, errorMessage, "CActiveMasternode::ManageStatus()")) {
+            notCapableReason = errorMessage;
             return;
+        }
```

Comment:
no need to create a new string variable `errorMessage`. We can directly pass `notCapableReason` (we just cleared it at line 48).

Also, better to change the logging inside `CheckDefaultPort` from `LogPrint` to `LogPrintf` as this is a permanent failure (and cannot really spam the logs).

Reply:
done